graphql/quizzes/insertCodingLanguageQuiz.ts

```diff
@@ -0,0 +1,11 @@
+import { gql } from "@apollo/client";
+
+export const INSERT_CODING_LANGUAGE_QUIZ_RESPONSE = gql`
+mutation insertCodingLanguageQuizResponse($name: String!, $email: String!) {
```

Comment:
did you test this mutation in hasura playground?

Reply:
yes, and it works!